### PR TITLE
doc: Extend supported board list in Features of nRF70 Series

### PR DESCRIPTION
**Pull request by richabp**
Extended supported board list in the
Features of nRF70 Series guide.

SHEL-1587

### CODE REVIEW

**Comment by richabp** [comment]
@mjd90 @bama-nordic : As suggested by @sachinthegreen, do we need to add nRF7001 DK in the list?

**Review by krish2718 — APPROVED** [review]

**Review by mjd90 — COMMENTED** [review]
Can we add an additional column in the table, e.g. "Companion module", between DK and PCA columns. The first row (7002 DK) would be N/A, the next 3 would be "nRF7002 EK" and the 4th would be "nRF7002 EB". We don't need the PCA number for these.

Once available, we should add links to the EK user guide (in the existing column).

File: doc/nrf/working_with_nrf/nrf70/features.rst [hunk 1]
```diff
@@ -29,25 +26,54 @@ The following figure illustrates the partitioning of the Wi-Fi stack between the
...
+     - nRF7002 EK
+     - PCA10090
+     - ``nrf9160dk_nrf9160_ns``
+     - | `Product Specification <nRF9160 Product Specification_>`_
+       | :ref:`Getting started <ug_nrf9160_gs>`
+       | `User Guide <nRF9160 DK Hardware_>`_
+   * - Thingy:53
+     - nRF7002 EB
+     - PCA20053
+     -
```
Comment (bama-nordic) [inline doc/nrf/working_with_nrf/nrf70/features.rst:71]: Please add Thingy53 board target - `thingy53_nrf5340_cpuapp`

**Review by bama-nordic — APPROVED** [review]

**Review by mjd90 — APPROVED** [review]
LGTM

**Pull request merged**